samples/package.json

```diff
@@ -0,0 +1,18 @@
+{
+  "name": "genkit-samples",
+  "private": true,
+  "scripts": {
+    "preinstall": "npx only-allow pnpm",
+    "build:js-samples": "for i in js-c*/; do cd $i && npm run setup && npm run build && cd ../; done"
```

Comment:
Consider using npm's `concurrently` helper lib for speed here. This is a perfect use case for it since all the samples can be built in isolation.